lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -80,7 +81,16 @@ export async function apiCall({ path, authToken, data }) {
     headers['Content-Type'] = 'application/json; charset=UTF-8';
   }
 
-  const result = await fetch(path, {
+  let query = '';
+  if (params) {
+    const urlParams = new URLSearchParams();
+    Object.entries(params).forEach(([name, value]) =>
+      urlParams.append(name, value)
+    );
+    query = `?${urlParams}`;
```

Comment:
Would it be helpful here to explicitly use the `toString` method, e.g. `?${urlParams.toString()}` ? It might make it more evident that we're working with an object here.

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/toString